README.md

```diff
@@ -1,33 +1,59 @@
 
-This repository hosts service interface definitions of Ethereum (2.0) APIs. These [protocol buffer](https://developers.google.com/protocol-buffers/) service definitions support gRPC and JSON over HTTP protocols.
+This repository hosts [Prysm](https://github.com/prysmaticlabs/prysm/)'s service interface definitions for the Ethereum 2.0 API. These [protocol buffer](https://developers.google.com/protocol-buffers/) service definitions support [gRPC](https://grpc.io/) as well as JSON over HTTP.
 
-## Motivation
-
-Standardizing common APIs is critical for the success of Ethereum 2.0. Developers may utilize these self-describing schemas to interact with Ethereum projects and having a central repository for service definitions may reduce the disparity between services across the Ethereum ecosystem. 
+### Need assistance?
+More indepth descriptions of each service are available in [this section](https://prysmaticlabs.gitbook.io/prysm/how-prysm-works/ethereum-2.0-public-api) of our official documentation. For more general information on the functionality of gRPC and protocol buffers, see the [gRPC guide](https://grpc.io/docs/guides/). If you still have questions, feel free to stop by either our [Discord](https://discord.gg/KSA7rPr) or [Gitter](https://gitter.im/prysmaticlabs/geth-sharding?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) and a member of the team or our community will be happy to assist you.
 
 ## Services
 
 | Package | Service | Version | Description |
 |---------|---------|---------|-------------|
-| eth | [BeaconChain](eth/v1alpha1/beacon_chain.proto#L36) | v1alpha1 | The beacon chain service can be used to access data relevant to the Ethereum 2.0 phase 0 beacon chain. |
-| eth | [Node](eth/v1alpha1/node.proto#L33) | v1alpha1 | General information about the Ethereum node itself, the services it supports, chain information, and node version. |
+| eth | [BeaconChain](eth/v1alpha1/beacon_chain.proto#L36) | v1alpha1 | This service is used to retrieve critical data relevant to the Ethereum 2.0 phase 0 beacon chain, including the most recent head block, current pending deposits, the chain state and more. |
+| eth | [Node](eth/v1alpha1/node.proto#L33) | v1alpha1 | The Node service returns information about the Ethereum node itself, including versioning and general information as well as network sync status and a list of services currently implemented on the node.
+| eth | [Validator](eth/v1alpha1/validator.proto) | v1alpha1 | This API provides the information a validator needs to retrieve throughout its lifecycle, including recieved assignments from the network, its current index in the state, as well the rewards and penalties that have been applied to it.
+| eth | [Attestation](eth/v1alpha1/attestation.proto) | v1alpha1 | The attestation service invokes the two functions required to create a cyptographic proof.
```

Comment:
The attestation isn't a service. A service will look like
```proto
service HelloService {
  rpc SayHello (HelloRequest) returns (HelloResponse);
}
```

Lets remove this line please.